components/system/components/ListEditor.js

```diff
@@ -3,7 +3,7 @@ import * as Constants from "~/common/constants";
 import * as SVG from "~/components/system/svg";
 
 import { css, keyframes } from "@emotion/react";
-import Draggable from "react-draggable";
+// import Draggable from "react-draggable";
```

Comment:
Why is this commented out? To fix this issue, you need to `npm install` after you rebase since @martinalong added some dependencies.

Reply:
Yep npm install worked!